tools/vms-generator/utils/utils.go

```diff
@@ -1259,7 +1259,7 @@ func GetVmCirrosClusterInstancetypeComputeSmall() *v1.VirtualMachine {
 	})
```

Comment:
Can you update `VmCirrosInstancetypeComputeSmall` to `VmCirrosClusterInstancetypeComputeSmall` as well?

Reply:
Done. Please, check if it is correct.